frontend/src/views/Crops/CropsModal.tsx

```diff
@@ -469,6 +485,69 @@ function CropsModal({
     return [{ id: Math.random(), reqN: remN, reqP2o5: remP2o5, reqK2o: remK2o }];
   }, [formData]);
 
+  // Check if N can be edited (Field vegetables with modifynitrogen = true AND calculations have been performed)
+  const isNEditable = selectedCropType?.modifynitrogen === true && calculationsPerformed;
+
+  // Dynamic columns for requirement table - make N editable for Field vegetables
+  const requirementColumns: GridColDef[] = useMemo(
+    () => [
+      {
+        field: 'reqN',
+        headerName: 'N',
+        width: 75,
+        minWidth: 75,
+        maxWidth: 200,
+        sortable: false,
+        resizable: false,
+        flex: 1,
+        ...(isNEditable && {
+          renderCell: (params) => (
```

Comment:
OOOOHHH now I see what you did to make this editable. You render a numberfield instead of the cell!! Very clever!!!